lib/Auth/Utils.php

```diff
@@ -129,7 +129,7 @@ public static function authenticatePI(array &$state, array $formParams): ?PIResp
         {
             try
             {
-                $response = $pi->validateCheck($username, $formParams["otp"], $transactionID);
+                $response = $pi->validateCheck($username, $formParams["otp"], $transactionID, $headersToForward);
```

Comment:
rename $headersToForward to $headers

Reply:
fixed [f2bb187](https://github.com/privacyidea/simplesamlphp-module-privacyidea/pull/198/commits/f2bb187f584e1a7a8c5c5fec38dee8d19a3e6b78)